lua/nvim-tundra/stylesheet/arctic.lua

```diff
@@ -1,5 +1,5 @@
-package.loaded['nvim-tundra.palette.arctic'] = nil
-local cp = require('nvim-tundra.palette.arctic')
+package.loaded["nvim-tundra.palette.arctic"] = nil
+local cp = require("nvim-tundra.palette.arctic")
```

Comment:
Small suggestion to revert unrelated changes:

```suggestion
package.loaded['nvim-tundra.palette.arctic'] = nil
local cp = require('nvim-tundra.palette.arctic')
```